pkg/nvml/mock/dgxh100/dgxh100.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dgxh100
+
+import (
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+	"github.com/NVIDIA/go-nvml/pkg/nvml/mock/shared"
+	"github.com/NVIDIA/go-nvml/pkg/nvml/mock/shared/gpus"
+)
+
+// Backwards compatible type aliases
+type Server = shared.Server
+type Device = shared.Device
+type GpuInstance = shared.GpuInstance
+type ComputeInstance = shared.ComputeInstance
+type CudaComputeCapability = shared.CudaComputeCapability
+
+func New() *Server {
+	return shared.NewServerFromConfig(shared.ServerConfig{
+		Config:            gpus.H100_SXM5_80GB,
+		GPUCount:          8,
```

Comment:
Question: Instead of specifying the count, why not pass in a list of devices? This will also make it simpler to handle heterogenous configs.